packages/cli/src/zostso/issue/command/Command.handler.ts

```diff
@@ -36,6 +37,14 @@ export default class Handler extends ZosTsoBaseHandler {
                 addressSpaceOptions: this.mTsoStart
             }
         );
+        const defProc = "IZUPROC";
+        if (params.arguments.logonProcedure && params.arguments.logonProcedure !== defProc && params.arguments.suppressStartupMessages) {
+            this.console.log(
+                chalk.yellow(
+                    "Warning: The logon procedure specified is not used when issuing a TSO command."
```

Comment:
Should we specifically say that this only occurs when suppressStartupMessages is used?